components/server/src/auth/auth-provider-service.ts

```diff
@@ -59,6 +59,7 @@ export class AuthProviderService {
             host: oap.host.toLowerCase(),
             verified: oap.status === "verified",
             builtin: false,
+            disallowLogin: !!oap.organizationId,
```

Comment:
nit: While I understand the reason it looks a bit strange to implement the same constraint in multiple places.

:cloud: Would it make sense to extract this into a method `allowsLogin(authProvider)` to capture that? :shrug:

Reply:
Adding this flag here is more of a cosmetically change, i.e. it's just used as a declaration, perhaps useful for logging and debugging. That said, when following the `disallowLogin` trails, I've learned that the actual check has vanished 💨 

The explicit checks around organizational Git Auth were added to autheticator.ts.